### PR TITLE
ZED-2076 Kolektivo Image Rework

### PR DESCRIPTION
**Pull request by Joshva02**

### CODE REVIEW

File: ios/celo/Images.xcassets/AppIcon.appiconset/Icon-App-60x60@3x.png
Comment (shottah) [inline ios/celo/Images.xcassets/AppIcon.appiconset/Icon-App-60x60@3x.png:1]: Check this file with the 60x60@3x.png file above.

File: ios/celo/Images.xcassets/splashBackground.imageset/Contents.json
Comment (shottah) [inline ios/celo/Images.xcassets/splashBackground.imageset/Contents.json:1]: Where is the splashBackground@1x.png changes?

**Review by shottah — APPROVED** [review]
Okay, looks good.

File: ios/celo/Images.xcassets/AppIcon.appiconset/Icon-App-60x60@3x.png
Comment (Joshva02) [reply ios/celo/Images.xcassets/AppIcon.appiconset/Icon-App-60x60@3x.png:1]: Resolved

**Pull request merged**